.demisto-sdk-report

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
what is it for?

Reply:
A mistake.
https://github.com/demisto/demisto-sdk/commit/ee5a29434a327ac2b7abe5bae6dbfb76dd406480